pkgs/development/python-modules/rethinkdb/default.nix

```
@@ -1,23 +1,26 @@
 { stdenv
 , buildPythonPackage
 , fetchPypi
+, python2
 }:
 
 buildPythonPackage rec {
   pname = "rethinkdb";
-  version = "2.4.4.post1";
+  version = "2.4.6";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "e1c1f8ad93bc1c6f2aaa73afc333c57d505d8cc08c437d78a5c1eb8dc4b7e8c2";
+    sha256 = "0x4xg2252k612g9i4n64rimvjy2znc78gmq73v8gbwwa94mjbd2f";
   };
 
   doCheck = false;
 
+  buildInputs = with python2.pkgs; [ six ];
```

in python-modules, the python packages are parameterized per python interpreter, don't bring in interpreter, rather individual packages
```suggestion
  buildInputs = [ six ];
```